package.json

```diff
@@ -44,7 +44,7 @@
     "redux": "3.5.1",
     "redux-immutable": "3.0.6",
     "redux-logger": "2.6.1",
-    "request": "2.71.0",
+    "request": "2.65.0",
```

Comment:
See also https://github.com/webpack/webpack/issues/138 and https://stackoverflow.com/questions/35236986/config-webpack-with-request